test/new-e2e/tests/apm/docker_test.go

```diff
@@ -48,7 +50,10 @@ func TestDockerFakeintakeSuiteUDS(t *testing.T) {
 		dockeragentparams.WithAgentServiceEnvVariable(
 			"STATSD_URL",
 			pulumi.String("unix:///var/run/datadog/dsd.socket")),
-	))
+	),
+		// The LoadBalancer uses https while the raw fakeintake uses http (incompatible with current config)
+		// Also, it can restart the fakeintake container
```

Comment:
Thanks for the comment - I think it can be improved still.

The `(incompatible with current config)` part is not clear. What exactly is not compatible and starting which version the compatibility was introduced?

`Also, it can restart the fakeintake container` can you please explain more what this means?

Reply:
Thanks for the review. I updated the description and moved this code to `dockerSuiteOpts`